go.mod

```diff
@@ -288,7 +296,7 @@ replace github.com/sercand/kuberesolver => github.com/sercand/kuberesolver v2.4.
 
 replace github.com/hpcloud/tail => github.com/grafana/tail v0.0.0-20201004203643-7aa4e4a91f03
 
-replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v36.2.0+incompatible
+replace github.com/Azure/azure-sdk-for-go/sdk/storage/azblob => github.com/MasslessParticle/azure-sdk-for-go/sdk/storage/azblob v0.2.1-0.20220131200443-9793e6f0cc65
```

Comment:
There is an open PR to msft from this fork that contains a deadlock fix. Point to it until the PR is merged

Reply:
Can you share the PR ?

Reply:
https://github.com/Azure/azure-sdk-for-go/pull/16937